core/src/config/api.ts

```diff
@@ -105,24 +113,21 @@ export const DEFAULT = {
 };
 
 export const test = {
-  general: (config) => {
+  [namespace]: () => {
     return {
-      developmentMode: true,
       startingChatRooms: {
         defaultRoom: {},
         otherRoom: {},
       },
-
       rpcTimeout: 3000,
     };
   },
 };
 
 export const production = {
-  general: (config) => {
+  [namespace]: () => {
     return {
       fileRequestLogLevel: "debug",
-      developmentMode: false,
```

Comment:
No idea what this is for, but noting that it's been removed from `production`

Reply:
I think that was a mistake - we don't want it (it's how the hot-reloading works)